### PR TITLE
Third branch

### PR DESCRIPTION
**Pull request by Jay2cof**
new request

### CODE REVIEW

File: lambda_function/lambda [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
```
Comment (fabianschmauder) [inline lambda_function/lambda:1]: This file is no tf file and is in a subfolder

File: Dynamodb/dynamondb.tf [hunk 2]
```diff
@@ -0,0 +1,15 @@
+resource "aws_dynamodb_table" "emailing-monitor" {
```
Comment (fabianschmauder) [inline Dynamodb/dynamondb.tf:1]: still in a subfolder

**Review by fabianschmauder — COMMENTED** [review]
move your terraform files out of the subfolder and rename a file to tf file

File: lambda_function/lambda (hunk 1 above)
Comment (Jay2cof) [reply lambda_function/lambda:1]: done

File: Dynamodb/dynamondb.tf (hunk 2 above)
Comment (Jay2cof) [reply Dynamodb/dynamondb.tf:1]: done

**Pull request merged**